includes/table/migrate-list.php

```diff
@@ -0,0 +1,127 @@
+<?php
+namespace Activitypub\Table;
+
+if ( ! \class_exists( '\WP_List_Table' ) ) {
+		require_once ABSPATH . 'wp-admin/includes/class-wp-list-table.php';
+}
+
+class Migrate_List extends \WP_List_Table {
+	public function get_columns() {
+		return array(
+			'post_author' => \__( 'Post Author (user_id)', 'activitypub' ),
+			'title' => \__( 'Post', 'activitypub' ),
+			'date' => \__( 'Publication Date', 'activitypub' ),
+			'comments' => \__( 'Comments', 'activitypub' ),
+			'migrate' => \__( 'Migrate', 'activitypub' ),
+		);
+	}
+
+	public function get_sortable_columns() {
+		return array();
+	}
+
+	public function prepare_items() {
+		$columns = $this->get_columns();
+		$hidden  = array( 'post_author', 'migrate' );
+		$sortable = $this->get_sortable_columns();
+
+		$this->process_action();
+		$this->_column_headers = array( $columns, $hidden, $sortable );
+
+		foreach ( \Activitypub\Tools\Posts::get_posts_to_migrate() as $post ) {
+			$this->items[] = array(
+				'post_author' => $post->post_author,
+				'title'      => \sprintf(
+					'<a href="%1s">%2s</a>',
+					\get_permalink( $post->ID ),
+					$post->post_title
+				),
+				'date'       => $post->post_date,
+				'comments'   => $post->comment_count,
+				'migrate'     => \get_post_meta( $post->ID, '_activitypub_permalink_compat', true ),
+			);
+		}
+
+		// pagination
+		$per_page = $this->get_items_per_page( 'elements_per_page', 10 );
+		$current_page = $this->get_pagenum();
+		$total_items = count( $this->items );
+		$table_data = array_slice( $this->items, ( ( $current_page - 1 ) * $per_page ), $per_page );
+		$this->set_pagination_args(
+			array(
+				'total_items' => $total_items,
+				'per_page'    => $per_page,
+				'total_pages' => ceil( $total_items / $per_page ),
+			)
+		);
+
+		// actions
+		if ( isset( $_REQUEST['_wpnonce'] ) ) {
+			$nonce = \esc_attr( $_REQUEST['_wpnonce'] );
+		}
+		// delete
+		if ( isset( $_REQUEST['action'] ) && 'activitypub_tools' === $_REQUEST['page'] && 'delete' === $_REQUEST['action'] ) {
```

Comment:
Thinking a bit further, I might remove the simple row_action delete, since there is also a Field to delete a federated URL